website/Vagrantfile

```diff
@@ -5,22 +5,22 @@ $script = <<SCRIPT
 sudo apt-get -y update
 
 # RVM/Ruby
-sudo apt-get -y install curl
-sudo apt-get -y install git
+sudo apt-get -qy install curl git libgmp3-dev
 gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
+# Install rvm and the latest version of ruby
 curl -sSL https://get.rvm.io | bash -s stable
 . ~/.bashrc
 . ~/.bash_profile
-rvm install 2.0.0
-rvm --default use 2.0.0
+rvm install ruby-2.2.2
+gem install bundler
 
 # Middleman deps
 cd /vagrant
-bundle
+make dev
 SCRIPT
 
 Vagrant.configure(2) do |config|
-  config.vm.box = "chef/ubuntu-12.04"
+  config.vm.box = "ubuntu/trusty64"
```

Comment:
Can we use a base box that supports VMware? The ubuntu builds [only support VirtualBox](https://atlas.hashicorp.com/ubuntu/boxes/trusty64). I think the chef ones are under [bento](https://atlas.hashicorp.com/bento/boxes/ubuntu-12.04) now and they support VBox, VMware, and Parallels.